<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionManager</class>
 <widget class="QWidget" name="ConnectionManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyLoad GUI - Connection Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connect to</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="hostLabel">
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="hostLineEdit">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="portSpinBox">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>64000</number>
        </property>
        <property name="value">
         <number>7227</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="useSSLLabel">
        <property name="text">
         <string>Use SSL:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="useSSLCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Login Data</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="userLabel">
        <property name="text">
         <string>User:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="userLineEdit">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="text">
         <string>password</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionReturnPressed">
   <property name="text">
    <string>returnPressed</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>userLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>actionReturnPressed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>actionReturnPressed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hostLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>actionReturnPressed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
